src/main/java/org/openstreetmap/atlas/checks/validation/points/LoneNodeCheck.java

```diff
@@ -0,0 +1,112 @@
+package org.openstreetmap.atlas.checks.validation.points;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.Area;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.LineItem;
+import org.openstreetmap.atlas.geography.atlas.items.Node;
+import org.openstreetmap.atlas.geography.atlas.items.Point;
+import org.openstreetmap.atlas.tags.HighwayTag;
+import org.openstreetmap.atlas.tags.RailwayTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check verifies lone {@link Node}s with highway tag that they follow the tagging principles.
```

Comment:
It would be confusing to mention Atlas Node here instead of Atlas Point. I suggest changing this to `This check verifies lone {@link Point}s`.